h/buildext.py

```diff
@@ -20,6 +21,8 @@
 
 import h
 from h._compat import urlparse
+from h.sidebar_app_config import app_config
```

Comment:
I'm not sure about the name of this module. I'd be inclined to call the module `h.sidebar`: `from h.sidebar import app_config` is just as clear and doesn't stutter like `from h.sidebar_app_config import app_config`.